src/components/Overlay/Overlay.tsx

```diff
@@ -185,13 +185,14 @@ class Overlay extends React.Component<IOverlayProps, IOverlayState, {}> {
 		return (
 			<Portal portalId={portalId}>
 				{isAnimated ? (
-					<ReactTransitionGroup
-						transitionName={cx('&')}
-						transitionEnterTimeout={300}
-						transitionLeaveTimeout={300}
-					>
-						{overlayElement}
-					</ReactTransitionGroup>
+					<TransitionGroup>
+						<CSSTransition
+							classNames={cx('&')}
+							timeout={300}
+						>
+							{overlayElement || <React.Fragment></React.Fragment>}
```

Comment:
The overlayElement might not exist so we need to provide a real react component, so I chose an empty fragment